app/models/reservation.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 class Reservation < ApplicationRecord
-  enum status: { created: 0, paid: 1, canceled: 2 }
+  enum status: { booked: 0, confirmed: 1, canceled: 2 }
   belongs_to :screening
-  belongs_to :user
+  belongs_to :user, optional: true
```

Comment:
I do not know your plans for the app. But reservation without user sounds strange. What kind of use cases do you see when reservation can be without user?

Reply:
Idea behind creating reservation without user is because we will have a ticket desks inside cinema where client can come and buy tickets instead of reserving them online so we don't have to have user to send tickets because I'm assuming that it would print tickets for them or something so that was the plan but we will be specifying it with Tobiasz tomorrow and in case that on the ticket desk there will be just some default employee logged in I'll change it